.circleci/config.yml

```diff
@@ -1298,11 +1298,11 @@ workflows:
           - images
           - check
 # keep this one disabled and enable only for development
-#    - e2e:
-#        <<: *commit_workflow_filters
-#        name: test/e2e-ipv6
-#        # custom parameters
-#        ipv6: true
+    - e2e:
+        <<: *commit_workflow_filters
+        name: test/e2e-ipv6
+        # custom parameters
+        ipv6: true
```

Comment:
do we want to enable this on commit? On one hand, it helps us catch things before merging to master, but this means twice as much CI runs?

Reply:
No I'll remove this but I wanted to have one run green with this.